ARCs/arc-0069.md

```diff
@@ -0,0 +1,173 @@
+            "type": "array",
+            "description": "Attributes following Open Sea's attributes format (https://docs.opensea.io/docs/metadata-standards#attributes)."
+        }
+    }
+}
+```
+The `standard` field is **REQUIRED**, all the other fields are **OPTIONAL**. But if provided, they **MUST** match the description in the JSON schema.
+
+The URI field (`external_url`) is defined similarly as the Asset URL parameter `au`.
+However, contrary to the Asset URL, it does not need to link to the digital media file.
+
+
+
+#### MIME Type
+
+In addition to including a MIME type in the `au` field of the asset, the JSON Metadata schema allows indication of the MIME type and subtype.
+
+
+
+#### Examples
+
+##### Basic Example
+
+An example of an ARC-69 JSON Metadata file for a song follows. The properties array proposes some **SUGGESTED** formatting for token-specific display properties and metadata.
+
+```json
+
+{   
+    "standard": "arc69",
+    "description": "arc69 theme song",
+    "external_url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+    "mime_type":"video/mp4",
+    "attributes": [
+        {"trait_type":"Bass",
+        "value":"Groovy"
+        },
+        {"trait_type":"Vibes",
+        "value":"Funky"
+        },
+        {"trait_type":"Overall",
+        "value":"Good stuff"
+        }
+    ]
+}
+```
+
+An example of possible ASA parameters would be:
+
+* *Asset Name*: `ARC69 theme song` for example.
+* *Unit Name*: `69TS` for example.
+* *Asset URL*: `ipfs://QmWS1VAdMD353A6SDk9wNyvkT14kyCiZrNDYAad4w1tKqT#v`
+* *Metadata Hash*: the 32 bytes of the SHA-256 digest of the high resolution media file.
+* *Total Number of Units*: 1
+* *Number of Digits after the Decimal Point*: 0
+
+#### Mutability
+
+##### Rendering
+
+Clients SHOULD render an ASA's latest ARC69 metadata. Clients MAY render an ASA's previous ARC69 metadata for changelogs or other historical features.
+
+##### Updating ARC69 metadata
+
+Managers MAY update an ASA's ARC69 metadata. To do so, they MUST send a new `acfg` transaction with the entire metadata represented as JSON in the transaction's `note` field.
+
+##### Making ARC69 metadata immutable
+
+Managers MAY make an ASA's ARC69 immutable. To do so, they MUST remove the ASA's manager address with an `acfg` transaction.
+
+## Rationale
+
+These conventions are heavily based on Ethereum Improvement Proposal [ERC-1155 Metadata URI JSON Schema](https://eips.ethereum.org/EIPS/eip-1155) to facilitate interoperobility. 
+
+The main differences are highlighted below:
+
+* Asset Name, Unit Name, and URL are specified in the ASA parameters. This allows applications to efficienty display meaningful information, even if they aren't aware of ARC-69 metadata.
+* MIME type fields are added to help clients know how to display the files pointed by URI.
+* All asset metadata is stored onchain.
+* Metadata can be either mutable or immutable.
+
+Valid JSON Metadata for ERC-1155 is valid JSON Metadata for ARC-69.

```

Comment:
Is it EIP-721?

Reply:
I suppose the only field we share with `EIP-721` is `description`. Removing this section is probably best